src/lib/web/routes/v1/rest/mavlink.rs

```diff
@@ -28,6 +34,42 @@ pub(crate) async fn post_mavlink(
     }
 }
 
+#[derive(Deserialize)]
+pub struct MessageInfo {
+    pub name: String,
+}
+
+pub(crate) async fn helper(name: Query<MessageInfo>) -> impl IntoResponse {
+    let message_name = name.0.name.to_ascii_uppercase();
+
+    let result = <mavlink::ardupilotmega::MavMessage as mavlink::Message>::message_id_from_name(
+        &message_name,
+    )
+    .and_then(|id| {
+        <mavlink::ardupilotmega::MavMessage as mavlink::Message>::default_message_from_id(id)
+    });
+
+    match result {
+        Ok(message) => {
+            let header = MAVLinkJSONHeader {
+                inner: Default::default(),
+                message_id: Some(mavlink::Message::message_id(&message)),
+            };
+            let json = serde_json::to_string_pretty(&MAVLinkJSON { header, message }).unwrap();
```

Comment:
```suggestion
            let json = serde_json::to_string_pretty(&MAVLinkJSON { header, message })?;
```

Reply:
this operation will never fail.

Reply:
then:
```suggestion
            let json = serde_json::to_string_pretty(&MAVLinkJSON { header, message }).expect("Should never fail here");
```

Reply:
Also, even if something never fails, it's still okay to use `?` if we can

Reply:
But this will generate an invalid HTTP response that's not a json.

Reply:
> But this will generate an invalid HTTP response that's not a json. Does not make more sense to add verbose code or a possible early return to a clear impossible operation. unwrap is there for a reason.

right, just use expect, then, so I won't get confused when reading the code in the future.

Reply:
This is a trivial operation, the type is known in compile time.